src/library/compiler/ir_interpreter.cpp

```diff
@@ -737,6 +733,35 @@ class interpreter {
             return interpreter(env).stub_m(args);
         }
     }
+
+    template <class ... Args>
+    static object * stub_N_aux(Args ... args) {
+        object * args2[] = { args... };
+        return interpreter::stub_m_aux(args2);
+    }
```

Comment:
@leodemoura Do tell me if you think that was a bad idea...

Reply:
Yes, I dislike it.

Reply:
and it's gone